package.json

```diff
@@ -78,20 +63,33 @@
     "wildemitter": "^1.2.0"
   },
   "devDependencies": {
-    "ava": "^0.20.0",
+    "@types/jest": "^24.0.15",
+    "@types/nock": "^10.0.3",
+    "@types/safe-json-stringify": "^1.1.0",
+    "@types/sinon": "^7.0.13",
```

Comment:
Jest has it's own mocking library built into it, so you shouldn't still need sinon

Reply:
Yeah, you are right. Initially I just wanted to get the tests working with as little changes as possible. I would like to remove `sinon` for sure. Maybe I will start doing that now that the main portion of the migration is complete.